annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -252,6 +252,17 @@ public MutationRecord createRecord(Map<String, String> mafLine) throws Exception
         return record;
     }
 
+     @Override
+    public boolean isHgvspNullClassifications(String variantClassification) {
+        return hgvspNullClassifications.contains(variantClassification);
+    }
+
+    @Override
+    public String getUrlForRecord(MutationRecord record, String isoformOverridesSource) {
+        String hgvsNotation = convertToHgvs(record);
+        return hgvsServiceUrl + hgvsNotation + "?" + isoformQueryParameter + "=" + isoformOverridesSource + "&"  + hotspotParameter + "=summary";
```

Comment:
let's also use this method for the `annotateRecord()` method [here](https://github.com/zheins/genome-nexus-annotation-client/blob/e97c657fc34deb16f67cc4c35017779b039f462c/annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java#L184)